drud/ddev
# Download WinNFSd and NSSM on demand fixes #2973, ddev_windows_installer detected as virus

## The Problem/Issue/Bug:
Virus tools more and more are recognizing WinNFSd and NSSM as malicious software and so these get not longer bundled to the installer but downloaded on demand during the installation to avoid issues.

## How this PR Solves The Problem:
WinNFSd and NSSM are now downloaded on demand, so only local virus scanners can complain.

## Manual Testing Instructions:
Remove the previous DDEV installation or in minimum remove the nssm.exe and WinNFSd.exe so see if it's correctly downloaded and installed with the new installer.

## Related Issue Link(s):
#2973 

## Release/Deployment notes:




<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3039"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
